Cargo.toml

```diff
@@ -4,6 +4,9 @@ version = "1.1.1"
 authors = ["Near Inc <hello@nearprotocol.com>"]
 edition = "2018"
 
+[profile.dev]
+debug = false
```

Comment:
Let's not bring this in every project explicitly. You can set it up locally using `~/.cargo/config` or `CARGO_PROFILE_DEV_DEBUG=false` environment variable.